### PR TITLE
Update messages of all commands

### LINKED ISSUE #150: Update messages displayed in popup

### PR DESCRIPTION
**Pull request by rachx**
Fixes #150 
@burnflare @INCENDE comment or make changes?

### CODE REVIEW

File: src/main/java/seedu/agendum/commons/core/Messages.java [hunk 1]
```diff
@@ -5,10 +5,11 @@
...
 public class Messages {
 
-    public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
-    public static final String MESSAGE_UNKNOWN_COMMAND_WITH_SUGGESTION = "Unknown command, did you mean '%1$s'?";
-    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_TASK_DISPLAYED_INDEX = "The task index provided is invalid";
-    public static final String MESSAGE_TASKS_LISTED_OVERVIEW = "%1$d tasks listed!";
+    public static final String MESSAGE_UNKNOWN_COMMAND = "We don't recognise this command";
+    public static final String MESSAGE_UNKNOWN_COMMAND_WITH_SUGGESTION = "Did you mean '%1$s'?";
+    public static final String MESSAGE_INVALID_COMMAND_FORMAT =
```
Comment (burnflare) [inline src/main/java/seedu/agendum/commons/core/Messages.java:10]: "Format" is not very user friendly. Just "we don't understand your input?"?

File: src/main/java/seedu/agendum/commons/core/Messages.java [hunk 2]
```diff
@@ -5,10 +5,11 @@
...
+            "We don't understand your command format. Try\n%1$s";
+    public static final String MESSAGE_INVALID_TASK_DISPLAYED_INDEX = "Hey, the task index given is invalid";
```
Comment (burnflare) [inline src/main/java/seedu/agendum/commons/core/Messages.java]: "Index" is not user friendly.

Comment (burnflare) [reply src/main/java/seedu/agendum/commons/core/Messages.java]: Same for all further usages of "index"

Comment (rachx) [reply src/main/java/seedu/agendum/commons/core/Messages.java]: id?

Comment (INCENDE) [reply src/main/java/seedu/agendum/commons/core/Messages.java]: I think index is fine though, can't think of any other word

Comment (burnflare) [reply src/main/java/seedu/agendum/commons/core/Messages.java]: id might be better. IMO easier to think that every "task has an id" than every "task has an index"

File: src/main/java/seedu/agendum/commons/core/Messages.java (hunk 1 above)
Comment (rachx) [reply src/main/java/seedu/agendum/commons/core/Messages.java:10]: addressed

File: src/main/java/seedu/agendum/commons/core/Messages.java (hunk 2 above)
Comment (rachx) [reply src/main/java/seedu/agendum/commons/core/Messages.java]: changed to id

**Comment by rachx** [comment]
image
Most of them (-Store/load) will look like that. Okay?
New lines are for the help window too.
To update alias and unalias too

**Comment by INCENDE** [comment]
Is it possible for the start-time to end-time be on the same line?

Other than that looks great!

File: src/main/java/seedu/agendum/logic/commands/DeleteCommand.java [hunk 3]
```diff
@@ -16,12 +16,14 @@
 
     // COMMAND_WORD, COMMAND_FORMAT, COMMAND_DESCRIPTION are for display in help window
     public static final String COMMAND_WORD = "delete";
-    public static final String COMMAND_FORMAT = "delete <index> \ndelete <index> <more-indexes>";
+    public static final String COMMAND_FORMAT = "delete <index>\n"
```
Comment (burnflare) [inline src/main/java/seedu/agendum/logic/commands/DeleteCommand.java]: this should be id too

File: src/main/java/seedu/agendum/logic/commands/DeleteCommand.java [hunk 4]
```diff
@@ -16,12 +16,14 @@
...
+                                              + "delete <index> <more-indexes>";
     public static final String COMMAND_DESCRIPTION = "delete task(s) from Agendum";
-    public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the tasks(s) identified by their index numbers used in the last task listing.\n"
-            + "Parameters: INDEX... (must be a positive number)\n"
-            + "Example: " + COMMAND_WORD + " 1 2 5-6";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " - "
+            + COMMAND_DESCRIPTION + "\n"
+            + COMMAND_FORMAT + "\n"
+            + "(The index must be a positive number)\n"
```
Comment (burnflare) [inline src/main/java/seedu/agendum/logic/commands/DeleteCommand.java]: this as well, lot's more places to change user facing index -> id.

**Comment by rachx** [comment]
@burnflare should be done

**Review by burnflare — APPROVED** [review]
Thanks, looks great

**Pull request merged**